main-app/app/scripts/game/services/game-api.js

```diff
@@ -3,10 +3,24 @@
     angular.module('Tombola.Games.Bingo90.Core')
         .service('GameApi',  ['GameProxy', function (gameProxy) {
             var me = this,
+                buyInData = {
+                    gameId: 1,
+                    userId: "drwho",
+                    balance: 19990
+                },
+                getCallData = {
+                    gameId: 1,
+                    userId: "drwho",
+                    balance: 19990,
+                    callnumber: 1
+                },
 
-                callApi = function(apiName, action, data, token){
+                //{gameId: config.game.gameId, card: config.game.card, user: player }
+
+            callApi = function(apiName, action, data, token){
                     gameProxy.callApi(apiName, action, data, token)
                         .then(function (data) {
+                            console.log(data);
                             //updateCallback(data);
```

Comment:
this sort of thing shouldn't appear in a real PR.
remove the console.log and put in todo: 
Danger is it would make it to live.